modules/ddl-and-loading/pages/system-and-language-basics.adoc

```diff
@@ -155,56 +168,65 @@ The catalog was reset and the graph store was cleared.
 
 === Summary
 
-The table below summaries the basic system commands introduced so far.
+The tables below summarize the basic commands introduced so far.
+
+==== GSQL commands
 
 |===
 | Command | Description
 
-| HELP[BASIC\|QUERY]
+| `HELP[BASIC\|QUERY]`
 | Display the help menu for all or a subset of the commands
 
-| LS
+| `LS`
 | Display the catalog, which records all the vertex types, edge types, graphs, queries, jobs, and session parameters that have been defined for the current active graph. See notes below concerning graph- and role-dependent visibility of the catalog.
 
-| BEGIN
+| `BEGIN`
 | Enter multi-line edit mode (only for console mode within the shell)
 
-| END
+| `END`
 | Finish multi-line edit mode and execute the multi-line block.
 
-| ABORT
+| `ABORT`
 | Abort multi-line edit mode and discard the multi-line block.
 
-| @file.gsql
+| `@file.gsql`
 | Run the gsql statements in the command file `file.gsql` from within the GSQL shell.
+|===
+
+==== System shell commands
 
-| `os$ gsql file.gsql`
+|===
+| Command | Description
+
+
+| `gsql file.gsql`
 | Run the gsql statements in the command file `file.gsql` from an operating system shell.
 
-| `os$ gsql 'command_string'`
+| `gsql 'command_string'`
 | Run a single gsql statement from the operating system shell.
 
-| `os$ gsql --reset`
+| `gsql --reset`
 | Clear the graph store and erase the dictionary.
 |===
 
 [NOTE]
 ====
-Notes on the LS command
+Notes on the `LS` command
 
-Starting with v1.2, the output of the LS command is sensitive to the user and the active graph:
+Starting with v1.2, the output of the `LS` command is sensitive to the user and the active graph:
```

Comment:
In the future, in general remove the version specifiers. All docs in this branch is supposed to describe this branch.
Especially when the version is this old, it makes more sense to remove them.